README.md

```diff
@@ -53,16 +53,37 @@ That JSON object is stored in a file in the [data](data/) directory, whose name
 
 Depending on the advancement of the underlying specification, the JSON object can have the following properties:
 * for W3C specifications that have started their Recommendation track progress, the `TR` property should point to the URL of latest version of the spec; this URL will be used to collect data about the spec (standardization status, Working Groups that produce it, editors draft, etc)
-* for specifications for which browser implementations are expected, the `impl` property is an object with the following optional properties:
+* for specifications for which browser implementations are expected, the `impl` property is an object with one or more of the following optional properties:
   * `caniuse`: the name of the feature in [Can I use](http://caniuse.com/) (the one that appears in the URL after `#feat=`)
   * `chromestatus`: the number used to identify features in [Chrome Platform Status](https://www.chromestatus.com/features) (the one that appears in the URL after `features/`)
   * `edgestatus`: the name used to identify features in [Microsoft Edge web platform features status and roadmap](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/) (the one that appears in the URL after `platform/status/`)
   * `webkitstatus`: the name used to identify features in [WebKit Feature Status](https://webkit.org/status/) (the one that appears in the URL after `status/#`)
+  * `other`: an object that describes known implementation status per user agent.
```

Comment:
Document the format of the object?

Reply:
OK, I completed the description and added an example. I did not mention the possibility to use an array at this stage. This does not seem needed as long as we don't need to convey more complete implementation information (e.g. which version of a user agent supports the feature)

Reply:
Oh, and I also added a warning that this mechanism should only be used as last resort when implementation data is not available elsewhere. I doubt we can do a good job at maintaining implementation status over time, and I would prefer us not to have to do it.